07-location.Rmd

```diff
@@ -18,14 +18,15 @@ library(tidyverse)
 
 ## Introduction
 
-This chapter demonstrates how the skills learned in the previous chapters can be applied to location analysis.
-Geomarketing (also called location analysis) is a broad field of research and commercial application, the aim of which is to decide the optimal location for new services.
-A standard example is where to locate a new shop: there are typically hundreds of possible locations in a city and location analysis can be used to decide where the shop will attract most visitors and, ultimately, make most profit.
-There are also many non-commercial applications that can use the technique for public benefit.
-A good example is where to locate new health services [@tomintz_geography_2008].
+This chapter demonstrates how the skills learned in Part I can be applied to a particular domain: location analysis (also called geomarketing).
+This is a broad field of research and commercial application, the aim of which is usually to decide the optimal location for new services.
+A typical example is where to locate a new shop.
+The aim here is to attract most visitors and, ultimately, make most profit.
+There are also many non-commercial applications that can use the technique for public benefit, for example where to locate new health services [@tomintz_geography_2008].
 
-Geomarketing is focused on people and where they are likely to spend their time and other resources.
-However, it has several links with ecology: animals and plants have needs which can be best met in optimal locations based on variables that change over space (called gradients in the ecological literature --- see e.g., @muenchow_review_2017<!--and chapter xx-->).
```

Comment:
Making this shorter because the term introduced, gradients, does not re-appear in the chapter. Easier to read without the brackets.

Reply:
Ok

Reply:
Yes - that's a vital point and set-up the eco chapter perfectly. The thinking is that introducing a new concept that is not used again in the chapter will dilute the point, not strengthen it, at this stage. We could talk about gradients of demand later in the chapter and re-instate it though but, re-reading it, I think there's already a lot to get one's head around. Hence the mission to simplify things and rely on links to further resources for advanced readers. Adding the link to the eco section will help with that so thanks for adding that in commented form.

Reply:
ok, agreed!